nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ConvertJSONToSQL.java

```diff
@@ -380,6 +383,10 @@ public void process(final InputStream in) throws IOException {
                 tableNameBuilder.append(tableName);
                 final String fqTableName = tableNameBuilder.toString();
 
+                if (USE_ATTR_TYPE.equals(statementType)) {
```

Comment:
What does this statement do?
`statementType = flowFile.getAttribute(STATEMENT_TYPE_ATTRIBUTE);`

Reply:
If the property of `Statement Type` is  ser to 'Use statement.type Attribute', we get the type of SQL statement from the incoming flowfile attribute